<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ovQDateDialog</class>
 <widget class="QDialog" name="ovQDateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>511</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QComboBox" name="monthComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="January">01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="February">02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="March">03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="April">04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="May">05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="June">06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="July">07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="August">08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="September">09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="October">10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="November">11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="December">12</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="dayComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="yearLabel">
      <property name="text">
       <string>Year (1+)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="monthLabel">
      <property name="text">
       <string>Month</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="dayLabel">
      <property name="text">
       <string>Day</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="yearLineEdit">
      <property name="inputMask">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="historicDateGridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="historicDateLabel">
      <property name="text">
       <string>Historical Dates</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="historicDateComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="historicDatePeriodComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>start</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>end</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="monarchDateGridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="monarchDateLabel">
      <property name="text">
       <string>Monarch Dates</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="monarchDateComboBox">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="monarchDatePeriodComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>start</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>end</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="infoLabel1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dates can either be entered</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>manually or by selecting one of</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>the pre-defined dates below.</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>yearLineEdit</tabstop>
  <tabstop>monthComboBox</tabstop>
  <tabstop>dayComboBox</tabstop>
  <tabstop>historicDateComboBox</tabstop>
  <tabstop>historicDatePeriodComboBox</tabstop>
  <tabstop>monarchDateComboBox</tabstop>
  <tabstop>monarchDatePeriodComboBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ovQDateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ovQDateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
